tests/MiraklMockedHttpClient.php

```diff
@@ -237,11 +237,17 @@ private function mockResponse(string $method, string $path, array $params, ?arra
 												$orderIds = explode(',', $params['order_ids']);
 										case isset($params['order_id']): // Service
 												$orderIds = $orderIds ?? (array) $params['order_id'];
+												if (count($orderIds) > 100) {
+													throw new Exception("Error Processing Request", 1);
```

Comment:
Using 400 instead of 1 is better here (this is the HTTP code).

Reply:
👍 makes sense